mozilla/activity-stream
# Bug 1566545 - Card UI spoc variants

Pre-req:
- Direct response promo/spoc with cta property set returned by Adzerk server https://spocs.getpocket.com/spocs

QA steps to test Link CTA variant
- By default a promo or spoc item should display CTA as link

QA steps to test Button CTA variant
- Set browser.newtabpage.activity-stream.discoverystream.config  "hardcode_layout" as false and layout_endpoint as 
"https://gist.githubusercontent.com/punamdahiya/da9f68852df22117c027c70eb99f3dbe/raw/f396f088233d981f39f843e4e1af9824d7165314/cta-button-4-spocs.json"

- Set browser.newtabpage.activity-stream.discoverystream.endpoints to https://,http://

A promo or spoc item should display CTA as button
